src/RageMath.h

```diff
@@ -5,7 +5,7 @@
 
 #include <vector>
 
-#define PI		(3.141592653589793f)
+constexpr float PI = 3.14159265358979323846f;
```

Comment:
Why are we doing a float, and then having like 20 decimal places lol.

Reply:
😅  It now matches the value from `extern\vorbis\lib\os.h`